tests/components/zwave_js/conftest.py

```diff
@@ -0,0 +1,59 @@
+"""Provide common Z-Wave JS fixtures."""
+import json
+from unittest.mock import patch
+
+import pytest
+from zwave_js_server.model.driver import Driver
+from zwave_js_server.model.node import Node
+
+from tests.common import MockConfigEntry, load_fixture
+
+
+@pytest.fixture(name="basic_data", scope="session")
+def basic_data_fixture():
+    """Load the basic fixture data."""
+    return load_fixture("zwave_js/basic_dump.txt").split("\n")
+
+
+@pytest.fixture(name="client")
+def mock_client_fixture(basic_data):
+    """Mock a client."""
+    state = json.loads(basic_data[0])["state"]
+    state["nodes"].clear()
+    with patch(
+        "homeassistant.components.zwave_js.ZwaveClient", autospec=True
+    ) as client_class:
+        driver = Driver(state)
+        client_class.return_value.driver = driver
+        yield client_class.return_value
+
+
+@pytest.fixture(name="multisensor_6")
+def multisensor_6_fixture(basic_data, client):
+    """Mock a multisensor 6 node."""
+    state = json.loads(basic_data[0])["state"]
+    for node_data in state["nodes"]:
```

Comment:
Preferably we break it out so we just have to load the data for multisensor 6 from a fixture. Otherwise we're going to load a lot of data for each test and only a subset is used.

Reply:
The data is only read from file once per session. But we do load it into json multiple times.

Reply:
I'll start cutting.